python/paddle/fluid/contrib/slim/quantization/imperative/qat.py

```diff
@@ -204,8 +201,18 @@ def _convert(model, *args):
                     dtype) if append_batch_size else raw_data.astype(dtype)
                 input_var = dygraph.to_variable(input_data)
                 input_vars.append(input_var)
-            prog_trans.get_output(_convert, model, *input_vars)
-        prog_trans.save_inference_model(dirname, feed, fetch)
+            outputs = prog_trans.get_output(model.forward, model, *input_vars)
+        input_spec = [input_vars[i] for i in feed]
+        configs = dygraph.jit.SaveLoadConfig()
+        configs.separate_params = True
+        if not isinstance(outputs, tuple):
```

Comment:
I'm not sure, should we also check "not instance(outputs, list)"

because you check a tuple but put a list here.

Reply:
Here I think `Layer.forward` only has `return x` or `return `x, y, ...` two format, so not check `list`

Reply:
done, txh